tests/formats_test/fstesting.py

```diff
@@ -87,9 +87,6 @@ def test_instantiation(self):
         self.assertEqual(an_fs.resizable, False)
 
         # sizes
-        expected_min_size = Size(0) if can_resize(an_fs) else an_fs._min_size
-        self.assertEqual(an_fs.min_size, expected_min_size)
```

Comment:
Should we not continue to verify this? The first expression would be simplified to `expected_min_size = an_fs._min_size` and the second line would remain as it is.

Reply:
Fixed.